server/src/controllers/projects/deleteProjectController.ts

```diff
@@ -0,0 +1,21 @@
+import { Response, Request, NextFunction } from 'express';
+import { deleteProjectQuery } from '../../database';
+
+const deleteProject = async (
+  req: Request,
+  res: Response,
+  next: NextFunction,
+) => {
+  const { id } = req.params;
+  const projectId = Number(id);
```

Comment:
instead of this line, you can convert string to number by add + when you pass an id as params to your query
`    await deleteProjectQuery(+projectId);`